px-context-browser.html

```diff
@@ -97,7 +98,7 @@
         on-iron-overlay-closed="_handleBrowserClosed">
       <px-context-browser-panel id="panel" slot="dropdown-content" has-header$="[[_isNotNull(active)]]" collapsed$="[[collapsed]]" show-loading-bar="[[loading]]">
         <px-context-browser-header id="header" slot="header" active$="[[_isNotNull(active)]]" label="[[_getItemProp(active, keys.label)]]" on-tap="_handleHeaderTapped" can-select="[[_isItemSelectable(active)]]" can-favorite="[[_showFavoritedAndItemFavoritable(showFavorited, active)]]" favorited="[[_isItemFavorited(active, favorited, favorited.*)]]" collapsed$="[[collapsed]]"></px-context-browser-header>
-        <px-context-browser-filter slot="filter" active$="[[showFilter]]" filter="{{filter}}" filtering="{{filtering}}"></px-context-browser-filter>
+        <px-context-browser-filter slot="filter" active$="[[showFilter]]" filter="{{filter}}" filtering="{{filtering}}" language="{{language}}" use-key-if-missing="{{useKeyIfMissing}}" resources="{{resources}}"></px-context-browser-filter>
```

Comment:
See comment in px-context-browser-filter.html, if we do that we can remove these properties and pass in placeholder.